src/functions/template-tags/general.php

```diff
@@ -1380,8 +1380,11 @@ function tribe_events_get_the_excerpt( $post = null, $allowed_html = null ) {
 			$excerpt = wp_trim_words( $excerpt, $excerpt_length, $excerpt_more );
 		}
 
-		// Remove all shortcode Content before removing HTML
-		if ( ! $allow_shortcode ) {
+		// If shortcode filter is enabled lets process them
+		if ( $allow_shortcode ) {
+			$excerpt = do_shortcode( $excerpt );
```

Comment:
I like that idea @barryhughes, and one more thing I think we could add a test to verify that we are not changing behaviors in the future.